static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -89,6 +89,17 @@
         {{#unless realm_allow_message_editing}}disabled="disabled"{{/unless}} />
     </div>
     <div class="input-group">
+      <label for="realm_message_retention_days"
+        id="id_realm_message_retention_days_label"
+        title="{{t 'If empty, messages are not moved to archive.' }}">
+        {{t 'Retention period for messages in days (empty for disable)' }}
```

Comment:
I'd like this to explicitly say something about data deletion, and be clearer about what "moved to archive" and "to disable" mean here, and linking to the help document (`retention`) would be good. I can take a crack at the phrasing after I edit `retention.md`.